### DISCUSSION TITLE
iFrame: combobox list options rendering outside of Modal

### DISCUSSION THREAD

**Discussion by and-lopes** [twilio-labs/paste#3392] [General] [answered]
The implementation details are a bit complicated, but we use this same custom country dropdown implementation in zen (no iframe) and it seems to work.

I'm trying to use it in an iframe (before we migrate it to zen) and it's rendering as follows:

### iframe
https://code.hq.twilio.com/numbers/numbers-console-frontend/pull/1250

https://github.com/twilio-labs/paste/assets/75698844/7d06f77f-cd45-4306-8504-71b240fbd351

### zen
![image](https://github.com/twilio-labs/paste/assets/75698844/e3b0404d-44b1-42f0-90fd-446eaf81d2d3)

Any ideas/pointers?

**Comment by SiTaggart** [comment]
Hi @and-lopes 

Try updating the version of Paste you are using. There was something we changed recently to the rendering of the Combobox Listbox (from absolute to fixed) that _might_ help.

Lets start there.

Thanks

**Reply by jmauriciosalasmoreno** [reply]
@SiTaggart it looks like this is [the fix](https://github.com/twilio-labs/paste/commit/abf6c19126a1e4fef5898c10354efe2141f27464) on Jul 14th and the latest release was on Jul 11th

I think the fix is not released yet, is it?

**Reply by SiTaggart** [reply]
Whoops! You're right, we'll be cutting a release today

**Comment by SiTaggart** [comment, marked as answer]
Hi @and-lopes 

To round this out, the fix I was referring to for Multiselect combobox is now in Production. But, as per our Slack conversations, this issue is likely to be just generally caused by an old version of Paste as our Single select combobox doesn't use absolute positioning, or some old styles on that particular page causing some issues, as you are using this component on a different page, in the same way, and it works just fine.

Hopefully you find a way to resolve this. Let us know how it goes

**Reply by and-lopes** [reply]
Linking this thread here as well for further context https://twilio.slack.com/archives/C030F220R8C/p1691627791584569

thanks for your help
